README.md

```diff
@@ -163,12 +162,14 @@ curl -i \
 
 ## Example
 
+Using the `--data-binary` and a /path/to/file can upload binary file types.
+
 ```shell
 curl -i \
     -X POST \
-    -H "Authorization: Basic abcdef==" \
+    -u 'USER:PASS' \
     -H "Content-Type: application/octet-stream" # This line
-    -d '{"hi":1}' \
+    --data-binary @/path/to/file \
```

Comment:
May also want to show a plaintext example, not just files. Thoughts?

Reply:
Yes, we should show plaintext as well as files — eg. a “Hello FISSION” string, a Markdown file, and JSON.

As I recall we can do interesting things with JSON + IPFS?

Maybe I’ll also do a HEROKU.md file that very specifically calls out how to work with what is essentially your own endpoint. Gets people used to Heroku even if they don’t access stuff from there. We only care about having a way to get people credentials.